3333.xml

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
                 <Type>Float</Type>
                 <RangeEnumeration>0..1</RangeEnumeration>
                 <Units>s</Units>
-                <Description>For shorter times of a fraction of a second (i.e. 0.23).</Description>
+                <Description>Fractional part of the time when sub-second precision is used.</Description>
```

Comment:
Maybe we want to keep an example for clarity? For example: `(e.g., 0.23 for 230 ms)`

Reply:
Added the example as discussed in the IPSO call on Oct 24th.